course-mw/course-actors-common/src/main/java/org/sunbird/learner/actors/coursebatch/dao/impl/UserCoursesDaoImpl.java

```diff
@@ -135,9 +135,12 @@ public List<String> getBatchParticipants(RequestContext requestContext, String b
   }
 
   @Override
-  public List<Map<String, Object>> listEnrolments(RequestContext requestContext, String userId) {
+  public List<Map<String, Object>> listEnrolments(RequestContext requestContext, String userId, List<String> courseIdList) {
```

Comment:
can we add params like primary properties for primary keys and fields for other fields? 
So that we can ensure Dao layer to be generic.